pkg/opssight/ctl_opssight.go

```diff
@@ -192,10 +192,11 @@ func (ctl *Ctl) CheckSpecFlags() error {
 
 // Constants for Default Specs
 const (
-	EmptySpec             string = "empty"
-	UpstreamSpec          string = "upstream"
-	DefaultSpec           string = "default"
-	DisabledBlackDuckSpec string = "disabledBlackDuck"
+	EmptySpec                string = "empty"
+	UpstreamSpec             string = "upstream"
+	DefaultSpec              string = "default"
+	DisabledBlackDuckSpec    string = "disabledBlackDuck"
+	DefaultBlackduckPassword string = "blackduck"
```

Comment:
This isn't a const for Default Specs. I would move it somewhere else or change the above comment.

Reply:
Done.

Reply:
@yashbhutwala Do we need to set the default password?? I think, we can remove it.